CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+CHANGES IN VERSION 1.9.0 (from 1.8.1)
+===================================
+
+Appium 1.9.0 is a feature release, comprising multiple updates.
+
+#### General
+* Full W3C Specification support.
+* Add full beta of [Espresso driver](https://github.com/appium/appium-espresso-driver)
+  for Android automation (used by specifying `automationName` capability to be
+  `"Espresso"`).
+* Add driver for [Samsung Tizen devices](https://github.com/Samsung/appium-tizen-driver)
+* Add `-image` find element strategy,
+* Fix `--async-trace` server argument, and rename as `--long-stacktrace`
+* Sample code has been moved into the main repository to aid in maintenance.
+* Fix status retrieval to speed up performance.
+
+#### iOS
+* Add support for Xcode 10 and iOS 12.
+* Add face id biometric support.
+* Fix retrieval of device time, and add optional `format` parameter.
+* Do not crash if there is no `idevicesyslog` when ending session.
+* Handle frames when page changes in Safari.
+* Add desired capabilities:
+  * `remoteDebugProxy` - port or Unix domain socket on which a proxy for the
+    remote debugger sits.
+  * `safariGarbageCollect` - turn on/off JS garbage collection in Safari.
+  * `showSafariNetworkLog` - print Safari network logs in the Appium server logs.
+* Fix handling of settings updates, so simulators are not restarted unnecessarily.
+* Allow pulling of files and folder from real devices.
+* Add `mobile: getContexts` execute function, to retrieve meta-information (title,
+  url, etc.) about available contexts.
+* Fix certificate retrieval and handling.
+* Fix cookie handling, to allow secure cookies.
+* Fix Safari timeout issues.
+* Add support to retrieve Safari network logs, as `safariNetwork` log type.
+
+#### Android
+* Update Chromedriver to v2.41.
+* Get Chrome version for Webviews in Android 7+, to find correct Chromedriver.
+* Make sure UiAutomator processes are cleaned up during test.
+* Fix handling of `autoWebview` capability.
+* New desired capabilities:
+  * `mjpegScreenshotUrl` - url to stream screenshots to.
```

Comment:
this is iOS, not Android

Reply:
also, It might be useful to mention, that now WDA starts broadcasting device/simulators screenshots in real time via MJPEG protocol if one connects to the default port 9100. Chrome, for example, supports this feature out of the box. Also screen recording has been rewritten to use this feature.

Reply:
For Android the screen recording has been rewritten in order to support recording of longer videos (up to 30 minutes now), but this requires FFMPEG utility to be available on the server machine (it is used to merge video chunks)

Reply:
https://github.com/appium/appium-uiautomator2-driver/commit/c9a8f67c1ceabd5d2ed631c6094c2febffdb322d

The cap doesn't exist for iOS (in `appium-ios-driver` or `appium-xcuitest-driver`).

Reply:
ah yes. I was thinking about `mjpegServerPort`

Reply:
@imurchie I'm pretty sure the cap also works for iOS, although maybe it's not one of the whitelisted caps?

Reply:
I can find no mention of it in
1. `appium-ios-driver` - https://github.com/appium/appium-ios-driver/search?q=mjpegScreenshotUrl&unscoped_q=mjpegScreenshotUrl
1. `appium-xcuitest-driver` - https://github.com/appium/appium-xcuitest-driver/search?q=mjpegScreenshotUrl&unscoped_q=mjpegScreenshotUrl
1. `appium-base-driver` - https://github.com/appium/appium-base-driver/search?q=mjpegScreenshotUrl&unscoped_q=mjpegScreenshotUrl

Not sure where else the functionality might be. Might be a mismatch with `mjpegServerPort` and `mjpegScreenshotUrl`.

Reply:
You're right. Hmm. Did I forget to do it in the xcuitest driver? Maybe no code changes are required. I'll have to test it.